### PR TITLE
Style

### PR DESCRIPTION
**Pull request by damiecode**

### CODE REVIEW

**Comment by samgaco** [comment]
### Hi @damiecode and @Sick-Carlito :hand: 

### You have done a phenomenal job. Everything is looking perfect :clap: but there are some points to address before we can give you the approval:

* In the **navigation bar**, the NY times logo should be a link and it should be also an embedded image.
* The section **More in Space & Cosmos** should contain links too. (The titles of the articles)
* There should be a **footer with a fixed** position (as you did with the navigation bar)
* Fix the errors given to you when uploading the index file to the [validator](https://validator.w3.org).
* You should be using **display flex or floats** to arrange the elements in the different sections (not grid)
* You should add the texts to this section ( you could use the image as the background of a div container and arrange the texts with flex for example).

> thissection

* You still have to implement those boxes at the right side for **Editor's picks** and **Sign up for Science Times.**

> rightside6

### Please :pray: work on these changes and submit back for another code review when you are done. You can do it! Keep up the good work! :muscle: :smiley:

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# New-York-Times
\ No newline at end of file
+# New-York-Times
+This Project shows features of the Newyork Times.
+We are cloning the main features of the article.
+
+#### [Assignment link]( https://www.theodinproject.com/courses/html5-and-css3/lessons/positioning-and-floating-elements)
+
```
Comment (stratospherique) [inline README.md:6]: - Add a live preview of your page

**Review by Ispirett — CHANGES_REQUESTED** [review]
Hey @damiecode @Sick-Carlito  Good Job Implementing the changes requested by @samgaco :+1: :100: 

However, there is a little more work to be done.

- [ ] This section here seems to be incomplete.

newyork times
 
#### Editor's Pick
- [ ] These sections should look more like the orginal site.
newyork2

##### Currently your pages look like this.

 
you new york

**Review by stratospherique — CHANGES_REQUESTED** [review]
## Hi Team :smile: . Great work, amazing output :clap: :clap: . However, some minor changes needs to be implemented in order for us to approve your project. 
- Your theory of inflation page is not properly centered.
- The elements just above that image are not centered too.
- More in Space & cosmos elements are aligned to the left. please fix them.
- Don't forget about the README file :arrow_down: 

### That's it guys. When finished, kindly resubmit another code review request. Keep up the good work :muscle: .

File: README.md (hunk 1 above)
Comment (damiecode) [reply README.md:6]: would we have to create the repository again? we didn't name the repository in a way it can be previewed live.

Comment (stratospherique) [reply README.md:6]: - No, that's not necessary. Just insert a link of your html file in the [githack](https://raw.githack.com) and that should give you a nice preview.

Comment (damiecode) [reply README.md:6]: ok thanks

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

File: style.css [hunk 2]
```diff
@@ -0,0 +1,507 @@
...
+}
+
+.body p {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+button {
+  height: 30px;
+  align-content: right;
```
Comment (Torres-ssf) [inline style.css]: [HINT]
Right is not a valid value for `align-content`
Check out [here](https://developer.mozilla.org/pt-BR/docs/Web/CSS/align-content)

File: style.css [hunk 3]
```diff
@@ -0,0 +1,507 @@
...
+}
+
+.Rick {
+  font-size: 11px;
+  margin-top: -10px;
+  color: grey;
+}
+
+.div1 #search {
+  flex: left;
```
Comment (Torres-ssf) [inline style.css]: [HINT]
Flex is not a value for flex. Check [this](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) link.

**Review by Torres-ssf — APPROVED** [review]
### Hi there team
Nice job implementing the required changes :+1: 
I'm giving this project a :+1: 
Go on and merge to master
Good luck with the next one :smile: 
Happy coding :computer:

**Pull request merged**